### PR TITLE
Bump version to 5.15.2 

### PR DESCRIPTION
**Pull request by Shkarlatov**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+        PYTHON_ALL_OUTPUT
+)
+
+message(
+    STATUS
+    "PYTHON_RESULT = ${PYTHON_RESULT}; "
+    "PYTHON_ALL_OUTPUT = ${PYTHON_ALL_OUTPUT}"
+)
+
+if((NOT PYTHON_RESULT EQUAL 0) OR (NOT PYTHON_ALL_OUTPUT MATCHES "2\.7"))
```
Comment (mr-bit) [inline CMakeLists.txt]: Add support for python 3.6.x too and update message below

File: rogii/build_common.cmake [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+    DEBUG_PATH
+    "${CMAKE_CURRENT_LIST_DIR}/../build/debug"
+)
+
+file(
+    MAKE_DIRECTORY
+    "${DEBUG_PATH}"
+)
+
+if(WIN32)
```
Comment (mr-bit) [inline rogii/build_common.cmake]: Remove and use -G Ninja below

File: rogii/version.cmake [hunk 3]
```diff
@@ -0,0 +1,5 @@
+SET(ROGII_QT_MAJOR_VERSION 5)
```
Comment (mr-bit) [inline rogii/version.cmake]: ROGII_MAJOR_VERSION, ROGII_MINOR_VERSION, ROGII_PATCH_VERSION and ROGII_PKG_VERSION for uniformity

**Review by mr-bit — CHANGES_REQUESTED** [review]
Small changes

**Review by chak3r — APPROVED** [review]

File: CMakeLists.txt [hunk 4]
```diff
@@ -0,0 +1,307 @@
...
+        PYTHON_ALL_OUTPUT
+)
+
+message(
+    STATUS
+    "PYTHON_RESULT = ${PYTHON_RESULT}; "
+    "PYTHON_ALL_OUTPUT = ${PYTHON_ALL_OUTPUT}"
+)
+
+if((NOT PYTHON_RESULT EQUAL 0))
```
Comment (mr-bit) [inline CMakeLists.txt]:
```
if((NOT PYTHON_RESULT EQUAL 0) OR ((NOT PYTHON_ALL_OUTPUT MATCHES "2\.7") AND (NOT PYTHON_ALL_OUTPUT MATCHES "3\.6")))
    message(
        FATAL_ERROR
        "Python 2.7.x or 3.6.x is not in the PATH."
    )
endif()
```

**Review by mr-bit — CHANGES_REQUESTED** [review]

**Review by mr-bit — APPROVED** [review]

**Pull request merged**